build/linux/installer/conf/td-agent-bit-prom-side-car.conf

```diff
@@ -0,0 +1,30 @@
+[SERVICE]
+    #Default service flush interval is 15 seconds
+    Flush         15
+    HTTP_Server   Off
+    Daemon        Off
+    storage.path  /var/opt/microsoft/docker-cimprov/state/flbstore/
+    storage.sync  normal
+    storage.checksum off
+    storage.backlog.mem_limit 10M
+    Log_Level     info
+    Parsers_File  /etc/opt/microsoft/docker-cimprov/azm-containers-parser.conf
+    Log_File      /var/opt/microsoft/docker-cimprov/log/fluent-bit.log
+
+[INPUT]
```

Comment:
[adding this here @ a  random place] - Telemetry differentiation dimension (for tgfmetricssentcount)